bats/tests/preferences/verify-paths.bats

```diff
@@ -0,0 +1,98 @@
+# Test case 30
+
+setup() {
+    load '../helpers/load'
+    if is_windows; then
+        skip "test not applicable on Windows"
+    fi
+    # Ensure subshells don't inherit a path that includes ~/.rd/bin
+    export PATH=$(echo "$PATH" | tr ':' '\n' | grep -v /.rd/bin | tr '\n' ':')
+}
+
+teardown_file() {
+    load '../helpers/load'
+    run rdctl shutdown
+    assert_nothing
+}
+
```

Comment:
This is already provided by the default implementation of `teardown_file()` now:

```suggestion
```

Reply:
And it's gone from this work